src/ui/win32/bindings/MemoryViewerControlBinding.cpp

```diff
@@ -231,6 +232,56 @@ bool MemoryViewerControlBinding::HandleNavigation(UINT nChar)
             }
             return true;
 
+        case 'C':
+            if (bControlHeld)
+            {
+                const auto& pEmulatorContext = ra::services::ServiceLocator::Get<ra::data::context::EmulatorContext>();
+                const auto iValue = pEmulatorContext.ReadMemory(m_pViewModel.GetAddress(), m_pViewModel.GetSize());
+                std::wstring sValue = ra::data::MemSizeFormat(iValue, m_pViewModel.GetSize(), MemFormat::Hex);
+
+                ra::services::ServiceLocator::Get<ra::services::IClipboard>().SetText(ra::Widen(sValue));
+            }
+            return true;
+
+        case 'V':
+            if (bControlHeld)
+            {
+                auto nAddress = m_pViewModel.GetAddress();
+                const auto& pEmulatorContext = ra::services::ServiceLocator::Get<ra::data::context::EmulatorContext>();
+                std::wstring sClipboardText = ra::services::ServiceLocator::Get<ra::services::IClipboard>().GetText();
+                auto n = m_pViewModel.GetSize();
+
+                if (sClipboardText.empty())
+                    return false;
+
+                // Check if the string is a valid hexadecimal value
+                for (wchar_t ch : sClipboardText)
+                    if (!iswxdigit(ch)) return false;
+
+                // Padding zeroes depending if shift is pressed (strict mode) or not (replace mode)
```

Comment:
The padding was made for the shift mode : in shift mode, the user was able to paste a long hexadecimal sequence and it was pasted and replaced value in memory.
Example : 12345 pasted into these words "1234 0000" resulted in "1234 5000"
 
I changed that in the last commit to be more strict to avoid confusion. 
The shift mode still let the user to paste long sequence but if they are not a multiple of the current size, the residual part will be converted without padding zeroes :
Example : 12345 pasted into these words "1234 0000" result now in "1234 0005"